_sass/_type.scss

```diff
@@ -1,3 +1,33 @@
+
+.page {
+  a:link,
+  a:visited {
```

Comment:
@rpeterman-gp This is where I added some styling to text links. Shouldn't affect the switcher links.